static/cdn/maintenance.html

```diff
@@ -1,208 +1,329 @@
-        <img src="http://scratch-maintenance.s3.amazonaws.com/scratch.png" style="margin:20px; padding:10px;">
-      </div>
-    </div>
+		<div class="page">
+			<div class="header-wrapper">
+				<div class="header-content">
+					<img src="http://scratch-maintenance.s3.amazonaws.com/logo_sm.png" width="80px" />
+				</div>
+			</div>
+			<div class="main">
+				<div class="content">
+					<h1>The Scratch Team is working hard to fix an issue with the Scratch website</h1>
+					<h2 class="reload" onclick="location.reload()">Try reloading the page in a few minutes</h2>
+					<h2 class="comment">Check <a href="https://twitter.com/scratch">@Scratch</a> for updates</h2>
+					<hr>
+					<div class="narrow">
+						<p>While the website is being fixed, you can use the Scratch app to make projects offline</p>
+					</div>
+					<button class="toggleShowing showing"><span>Download Scratch app</span></button>
+					<div class="grid-wrapper hidden">
+					  <div class="grid-content">
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/windows-logo.svg" width="28px" />
+									</div>
+									<div class="os-name-text">Windows</div>
+								</div>
+								<div class="os-version">Windows 10+</div>
+								<div class="download-link-row">
+									<div class="store-img">
+										<a href="https://www.microsoft.com/store/apps/9pfgj25jl6x3?cid=storebadge&ocid=badge">
+											<img src="http://scratch-maintenance.s3.amazonaws.com/windows-store-badge.svg" />
+										</a>
+									</div>
+									<div class="vertical-separator-left"></div>
+									<div class="vertical-separator-right"></div>
+									<div class="download-link"><a href="https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe">Download</a></div>
+								</div>
+							</div>
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/apple-logo.svg" width="28px" />
+									</div>
+									<div class="os-name-text">macOS</div>
+								</div>
+								<div class="os-version">macOS 10.13+</div>
+								<div class="download-link-row">
+									<div class="store-img">
+										<a href="https://apps.apple.com/us/app/scratch-desktop/id1446785996?mt=12">
+											<img src="http://scratch-maintenance.s3.amazonaws.com/mac-store-badge.svg" />
+										</a>
+									</div>
+									<div class="vertical-separator-left"></div>
+									<div class="vertical-separator-right"></div>
+									<div class="download-link"><a href="https://downloads.scratch.mit.edu/desktop/Scratch.dmg">Download</a></div>
+								</div>
+							</div>
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/chromeos-logo.svg" width="26px" />
```

Comment:
Perhaps it's intentional, but I notice this image is 26px instead of 28px like the others.

Reply:
I thought it looked too big at 28px, but now I don't know what I was thinking! Changing back to 28px